web/themes/custom/pineapple/templates/content/node--author--full.html.twig

```diff
@@ -0,0 +1,101 @@
+{#
```

Comment:
Some general notes:
- create url aliases for the author pages, the structure should be as follows **authors/name-of-author**
- Add breadcrumbs to the authors and also individual author pages